lib/sift/value_parser.rb

```diff
@@ -33,7 +33,9 @@ def parse_json_and_values
       return parsed_jsonb if parsed_jsonb.is_a?(Array) || parsed_jsonb.is_a?(String)
 
       parsed_jsonb.each_with_object({}) do |key_value, hash|
-        hash[key_value.first] = parse_json(key_value.last.to_s)
+        key   = key_value.first
+        value = key_value.last
+        hash[key] = value.is_a?(String) ? parse_json(value) : value
```

Comment:
is this change to allow nil to be passed as a param?

Reply:
This is a related change for this case: 

When the JSOB filter is not being sent as a String: `filters[custom_field_8]=[false, null]` instead of `"[false, null]"`, the Parsed value is: 
```
{
  "custom_field_8" => [false, nil]
}
```
Parsing each value not necessary when the value is not a String, specifically `[false, nil]` will not be parsed correctly because `nil` is not valid.